src/org/usfirst/frc/team4915/stronghold/subsystems/Intake_Launcher.java

```diff
@@ -0,0 +1,69 @@
+package org.usfirst.frc.team4915.stronghold.subsystems;
+
+import edu.wpi.first.wpilibj.CANTalon;
+import edu.wpi.first.wpilibj.CANTalon.TalonControlMode;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.usfirst.frc.team4915.stronghold.RobotMap;
+
+public class Intake_Launcher extends Subsystem {
+
+    // Ranges -1 to 1, negative values are reverse direction
+    // Negative speed indicates a wheel spinning inwards and positive speed
+    // indicates a wheel spinning outwards.
+    public static final double INTAKE_SPEED = -0.5;
+    public static final double LAUNCH_SPEED = 1.0;
+    public static final double ZERO_SPEED = 0.0;
+    public static final double ELEVATOR_SPEED = 0.25;
+    public static final double ELEVATOR_MAX_HEIGHT = 0.0;
+    public static final double ELEVATOR_MIN_HEIGHT = 0.0;
+
+    // left and right are determined when standing behind the robot
+
+    // These motors collect and shoot the ball
+    public CANTalon intakeLeftMotor = RobotMap.intakeLeftMotor;
```

Comment:
Would it maybe be easier to just refer to `RobotMap.intakeLeftMotor` rather than assign it to a new `intakeLeftMotor` object in this class?